modules/effects/compat/rollup.config.js

```diff
@@ -0,0 +1,12 @@
+export default {
```

Comment:
I copied those from `testing` but I have not idea whether they are correct in the context of compat.